src/registrar/admin.py

```diff
@@ -103,6 +103,36 @@ class MyUserAdmin(BaseUserAdmin):
 
     inlines = [UserContactInline]
 
+    list_display = (
+        "email",
+        "first_name",
+        "last_name",
+        "is_staff",
+        "is_superuser",
+        "status",
+    )
+
+    fieldsets = (
+        (
+            None,
+            {"fields": ("username", "password", "status")},
+        ),  # Add the 'status' field here
```

Comment:
NIT: Can remove this comment I think since you've added it above (unless it's a holder for something else, sorry!)